src/patternfly/components/List/docs/code.md

```diff
@@ -1,11 +1,13 @@
 ## Overview
 
-Typically used with a modifier class.
+The list component should be used with a modifier to determine its style as a `<ul>` or `<ol>`. `.pf-c-list__item` is required. Lists can be nested up to any level.
 
 ## Usage
 
 | Class | Applied To | Outcome |
 | -- | -- | -- |
-| `.pf-c-list` | `<ul>` |  Initiates a list with the same styling as lists in the content component. **Required** |
-| `.pf-m-inline` | `.pf-c-list` |  Modifies for inline list style. |
-| `.pf-m-grid` | `.pf-c-list` |  Modifies for grid list style. |
+| `.pf-c-list` | `<ul>, <ol>` | Initiates a list. Must be used with a modifier. **Required** |
+| `.pf-c-list__item` | `<li>` | Initiates a list item. **Required** |
```

Comment:
Why wouldn't you just style the `li`?

Reply:
I created `.pf-c-list__item` so that in the event that a modifier or attribute needs to be added to the list-item it can added in the hbs file. Do you think its safe to say that it won't need a modifier and I should just use `<li>` ?

Reply:
Ah it's because you have a `ul` as a child of the `ol`. Only a `li` can be a child of either a `ul` or `ol`. If you nest a list in a list, it needs to be nested in the `li`, not the `ol`, or `ul`.

Reply:
ok that makes sense!